getting_started/configuration_file.md

```diff
@@ -6,22 +6,110 @@ compiler, formatter and linter.
 The configuration file supports `.json` and `.jsonc` extensions.
 [Since v1.18](https://deno.com/blog/v1.18#auto-discovery-of-the-config-file),
 Deno will automatically detect `deno.json` or `deno.jsonc` configuration file if
-it's in your current working directory (or parent directories). To manually tell
-Deno to use a specific configuration file pass `--config path/to/file.json`
-flag.
-
-> ⚠️ Starting with Deno v1.22 you can disable automatic detection of the
-> configuration file, by passing `--no-config`.
-
-Note that using a configuration file is not required now, and will not be
```

Comment:
I guess this is now gone :D

Reply:
It's not gone - it's just not so important that it needs to be here.